capstone/src/test/resources/com/google/univiz/scorecard/scorecard.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Note that you'll probably need to introduce another Autovalue class to handle this full response:
* `ScorecardData` corresponds to a single `result`.
* The new Autovalue would look something like:
```
@AutoValue
SearchResponse {
  abstract List<ScorecardData> results;
}
```
* This can happen in a follow up PR

Reply:
is this so that it doesnt read the ```metadata``` array?